src/geom/polygon.jl

```diff
@@ -69,20 +70,25 @@ function render(geom::PolygonGeometry, theme::Gadfly.Theme, aes::Gadfly.Aestheti
     colors = Vector{CT}(undef, n)
     line_styles = Vector{LST}(undef, n)
     linestyle_palette_length = length(theme.line_style)
+    alphas = Vector{Float64}(undef, n)
+    alpha_discrete  = AT <: Int
+
     for (k,g) in enumerate(ug)
         i = groups.==[g]
         polys[k] = polygon_points(aes_x[i], aes_y[i], geom.preserve_order)
         colors[k] = first(aes_color[i])
-        line_styles[k] = mod1(first(aes_linestyle[i]), linestyle_palette_length) 
+        line_styles[k] = mod1(first(aes_linestyle[i]), linestyle_palette_length)
+        alphas[k] = first(alpha_discrete ? theme.alphas[aes_alpha[i]] : aes_alpha[i])
```

Comment:
what git weirdness is causing the `line_style[k]` line to be highlight as a change?

Reply:
Weird - no idea.